### PR TITLE
Nodejs week3

### PR DESCRIPTION
**Pull request by VioMNE**
Here is my SQL injection demonstration :) 

https://user-images.githubusercontent.com/103515244/220100206-ffea67a9-eeca-400a-ad68-12cad9b53bde.mp4

### CODE REVIEW

File: nodejs/week3/app.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+apiRouter.use("/contacts", contactsAPIRouter);
+
+contactsAPIRouter.get("/", async (req, res) => {
+  let query = db.select("*").from("contacts");
+
+  if ("sort" in req.query) {
+    const orderBy = req.query.sort.toString();
+      if (orderBy.length > 0) {
+      query = query.orderByRaw('??', [orderBy]);
+    }
```
Comment (Divyajg) [inline nodejs/week3/app.js:34]: Can you explain how the above modification prevents the SQL injection?

**Review by Divyajg — COMMENTED** [review]
One of the good articles try reading,
https://planetscale.com/blog/how-to-prevent-sql-injection-attacks-in-node-js

**Review by Divyajg — APPROVED** [review]
Good work 👍

File: nodejs/week3/app.js (hunk 1 above)
Comment (VioMNE) [reply nodejs/week3/app.js:34]: Thank you so much for your time to review my homework :) Have a lovely day :)

**Pull request merged**